crates/matrix-sdk-ui/src/room_list_service/room.rs

```diff
@@ -154,22 +154,32 @@ impl Room {
     /// contains a local event. Otherwise, it comes from the cache. This method
     /// does not fetch any events or calculate anything — if it's not
     /// already available, we return `None`.
+    ///
+    /// Reminder: this method also returns `None` is the latest event is not
+    /// suitable for use in a message preview.
     pub async fn latest_event(&self) -> Option<EventTimelineItem> {
         // Look for a local event in the `Timeline`.
```

Comment:
nit: this comment needs a change too, now it's local or remote.

Reply:
Indeed, thanks.